src/services/pdf.service.js

```diff
@@ -129,13 +129,7 @@ const createSingleSection = (title, sectionData, i18n) => {
       (key == "establishment_email" || key == "operator_email") &&
       answer.length >= 35
     ) {
-      var answerWithBreak = "";
-      for (var i = 0; i < answer.length; ++i) {
-        if (answer[i] == "@") {
-          answerWithBreak += "\n";
-        }
-        answerWithBreak += answer[i];
-      }
+      var answerWithBreak = answer.replace("@", "\n");
```

Comment:
my bad: you need to replace with "@\n" as you need to keep the @

Reply:
addressed. don't know how I missed it too as I tested it out also aha